.changeset/plenty-schools-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/lab": minor
+---
+
+Remove `onClose` prop from `Overlay`, onOpenChange prop required in controlled state
```

Comment:
Does this mean uncontrolled can't be used?

Reply:
The overlay manages the open state by default in an uncontrolled state - if the user wants something additional to happen when the close button is clicked then yes the overlay would have to be in a controlled state

Reply:
We can remove this:

```suggestion
Removed `onClose` prop from `Overlay`. `onOpenChange` is called for events that open/close the overlay.
```